charts/k8s-watcher/README.md

```diff
@@ -94,115 +94,116 @@ To install the chart directly with kubectl, use the manifests located in `./kube
-| `network_mapper.debug`   | Enable debug mode for network mapper | `false` |
-| `network_mapper.mapper.resources`  | Set network mapper resources | `{}` |
-| `network_mapper.sniffer.resources` | Set resources for network mapper sniffer | `{}` |
-| `daemon.tolerations`                               | Tolerations for daemonset pod assignment - Example:   --set 'daemon.tolerations[0].key=my-key' --set 'daemon.tolerations[0].operator=Equal' --set 'daemon.tolerations[0].value=my-value' --set 'daemon.tolerations[0].effect=NoSchedule'                                                                                                                                                                         | `[]`                                |
-| `supervisor.enabled`                                  | Enables supervisor sidecar (monitoring the agent)                                                                                                                                                                                    | `false`                             |
+| Parameter                                                  | Description                                                                                                                                                                                                                                               | Default                             |
+|------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------|
+| `apiKey`                                                   | Komodor kubernetes api key (required if `existingSecret` not specified)                                                                                                                                                                                   | ``                                  |
+| `existingSecret`                                           | Existing kubernetes secret resource containing Komodor kubernetes apiKey (required if `apiKey` not specified)                                                                                                                                             | ``                                  |
+| `tags`                                                     | Tags the agent in order to identify it based on `key:value` properties separated by semicolon (`;`) - example: `--set tags="env:staging;team:product-a"` --- Can also be set in the values under `watcher.tags` as a dictionary of key:value strings      | ``                                  |
```

Comment:
maybe add to the description
```Tags the agent in order to identify it and the cluster based on....```

Reply:
Ehmm.. that's exactly what is there already? :)